wai-extra/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.19
+
+* Add a new function basicAuth', which passes request to the CheckCreds variant.
```

Comment:
I'm not sure what "variant" means in this case, can you clarify? Perhaps this should be "argument"?